tests/monster_test_generated.rs

```diff
@@ -292,6 +300,21 @@ impl flatbuffers::Push for Race {
     }
 }
 
+impl TryFrom<i8> for Race {
+    type Error = flatbuffers::ConvertError<i8>;
+
+    #[inline]
+    fn try_from(value: i8) -> Result<Self, Self::Error> {
+        match value {
+          -1 => Ok(Race::None),
+          0 => Ok(Race::Human),
+          1 => Ok(Race::Dwarf),
+          2 => Ok(Race::Elf),
+          _ => Err(Self::Error::InvalidValue(value))
+        }
+    }
+}
+
 #[allow(non_camel_case_types)]
 pub const ENUM_VALUES_RACE: [Race; 4] = [
```

Comment:
Please deprecate `ENUM_VALUES_RACE`, `ENUM_NAMES_RACE`, and `enum_name_race`, and make them associated constants of `Race`; or leave a TODO to do so in a future pull

Reply:
These might be useful. Are there reasons to deprecate them?

Reply:
I think its more idiomatic for these to be associated constants with namespaced names rather than old-c-style prefixy names.
e.g. `Race::VALUES` instead of `ENUM_VALUES_RACE`.

While we're renaming things we should give people some opportunity to refactor their code, i.e. deprecate and give a deletion timeline.